### ISSUE TITLE
Interview Summarizer | HacktoberFest 2023 

### ISSUE THREAD

**Issue by Mohit030802** [Nayaker/AlgoListed#108]
Hi folks 🤠,
### OVERVIEW: 
I'd like to suggest the addition of a new function to the Interview Topic Summarizer, an innovative application developed to help people prepare for technical interviews. The proposed feature aims to provide users with company-specific statistics on the number of questions asked and their difficulty level in specific topics such as Data Structures and Algorithms (DSA), Computer Networks (CN), Operating Systems (OS), Database Management Systems (DBMS), and Object-Oriented Programming (OOPS).
### PROBLEM STATEMENT:
Preparing for technical interviews is an important step in getting a job in the computer business. However, candidates frequently lack a vision of the exact themes on which they should focus during interviews with various companies. Each employer may stress different areas of knowledge, making it difficult for candidates to appropriately prioritize their preparation.
### PROPOSAL:
The proposed feature for the Interview Topic Summarizer seeks to address this challenge by offering users company-wise statistics for their selected topics. Here are the key functionalities:

**Key Features**
1. Topic Selection: Users can choose from a list of technical topics, such as DSA, CN, OS, DBMS, and OOPS.

2. Company-wise Statistics: After selecting a topic, users can view a graph or chart that displays company-wise statistics for that specific topic. This graph will include information on the number of questions asked by each company for the chosen topic.

3. Difficulty Level: The statistics can also categorize questions by difficulty level (e.g., easy, medium, hard) to help users gauge the complexity of questions asked by different companies.

### CALL TO ACTION:

I respectfully request that the creators of the Interview Topic Summarizer take this suggestion for the addition of the company-specific statistics function into consideration. This addition has the potential to greatly improve the tool's functionality and offer users useful insights for more targeted interview preparation.

Thank you for your time and consideration, and I look forward to hearing your thoughts on the implementation of this feature.

### ADDITIONAL INFORMATION:

I have attached a Figma file for reference : [Figma](https://www.figma.com/file/unXGjYzn0lZCLlyCXvwJMx/Algolisted--Interview-Statistics?type=design&node-id=0%3A1&mode=design&t=bciX6Hu9nEYvAoxt-1)

**Comment by inbuilt-aura** [comment]
@Mohit030802 do we have to work on every part or work on individual component as this project is huge.I would love to contribute though

**Comment by Mohit030802** [comment]
Hi @inbuilt-aura, you can work on the UI part first, and once the UI is complete, we can make the graphs dynamic using the data we have.

**Comment by inbuilt-aura** [comment]
@Mohit030802, i am still working on the ui broo,plzz open it

**Comment by inbuilt-aura** [comment]
@Mohit030802 ,thanks

**Comment by inbuilt-aura** [comment]
@Mohit030802 THERE IS NO package.json file here

**Comment by Mohit030802** [comment]
@Nayaker  Solved this issue right @inbuilt-aura?

**Comment by inbuilt-aura** [comment]
@Mohit030802 got it.Thanks

**Comment by Mohit030802** [comment]
HI @inbuilt-aura any updates?

**Comment by inbuilt-aura** [comment]
Yeah @Mohit030802, sorry about not updating you actually my laptop was damaged due to some issues,i have almost done with ui part but then that happened. I ordered new laptop which will be delivered on 31st.I hope i will be able to recover that.If not, anyone else want can contribute to this,if not I will again contribute once i am doen with setting up my laptop.So highly apologise for that.

**Comment by Mohit030802** [comment]
No worries @inbuilt-aura. You still can work on this once your laptop arrives👍.
